README.md

```diff
@@ -1 +1,38 @@
-# A-setup-and-mobile-version-skeleton
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Setup-and-mobile-version-skeleton
```

Comment:
Please add some description here like what's your project and what you'll be doing in it 👍🏻